.ado/templates/prep-and-pack-nuget.yml

```diff
@@ -49,29 +49,30 @@ steps:
         Write-Host "##vso[task.setvariable variable=baseConfiguration]${{ parameters.slices[0].configuration }}"
       displayName: Set base slice to ${{ parameters.slices[0].platform }}.${{ parameters.slices[0].configuration }}
       
-  - ${{ each slice in parameters.slices }}:
-    - task: DownloadPipelineArtifact@2
-      displayName: 'Download ${{ parameters.artifactName }}.${{ slice.platform }}.${{ slice.configuration }} Artifact'
-      inputs:
-        artifact: ${{ parameters.artifactName }}.${{ slice.platform }}.${{ slice.configuration }}
-        path: $(System.DefaultWorkingDirectory)/ReactWindows/${{ slice.platform }}/${{ slice.configuration }}
+  - ${{ each artifact in parameters.artifacts }}:
+    - ${{ each slice in parameters.slices }}:
```

Comment:
This isn't right for publish either, since we do not pass slices into the template there...